packages/embark-ui/src/containers/ContractOverviewContainer.js

```diff
@@ -2,23 +2,18 @@ import React, {Component} from 'react';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
 
-import {contractProfile as contractProfileAction, contractFunction as contractFunctionAction} from '../actions';
+import {contractFunction as contractFunctionAction} from '../actions';
```

Comment:
Should we also remove the `contractProfile` action from actions or is it used in other places?

Reply:
I don't think it's used anywhere else, but I'd keep it for now, as it makes it easier for us to add/add back a profile page for contracts.